src/sssom/util.py

```diff
@@ -1601,3 +1589,18 @@ def invert_mappings(
 def _invert_column_names(column_names: list, columns_invert_map: dict) -> dict:
     """Return a dictionary for column renames in pandas DataFrame."""
     return {x: columns_invert_map[x] for x in column_names}
+
+
+def safe_compress(uri: str, converter: Converter) -> str:
+    """Parse a CURIE from an IRI.
+
+    :param uri: The URI to parse. If this is already a CURIE, return directly.
+    :param converter: Converter used for compression
+    :return: A CURIE
+    """
+    if not is_curie(uri):
+        return converter.compress_strict(uri)
+    rv = converter.standardize_curie(uri)
+    if rv is None:
+        raise ValueError
```

Comment:
Is there a reason the ValueError is not a bit more verbose?

Reply:
Fixed in 7e19639